LayoutTests/TestExpectations

```diff
@@ -4211,8 +4211,6 @@ imported/w3c/web-platform-tests/css/css-text-decor/text-emphasis-position-under-
 imported/w3c/web-platform-tests/css/css-text-decor/text-emphasis-position-under-right-002.xht [ ImageOnlyFailure ]
 imported/w3c/web-platform-tests/css/css-text-decor/text-emphasis-color-001.xht [ ImageOnlyFailure ]
 imported/w3c/web-platform-tests/css/css-text-decor/text-emphasis-style-002.html [ ImageOnlyFailure ]
-imported/w3c/web-platform-tests/css/css-text-decor/text-emphasis-punctuation-1.html [ ImageOnlyFailure ]
-imported/w3c/web-platform-tests/css/css-text-decor/text-emphasis-punctuation-2.html [ ImageOnlyFailure ]
 imported/w3c/web-platform-tests/css/css-text-decor/text-emphasis-punctuation-3.html [ ImageOnlyFailure ]
```

Comment:
There's a few allowed punctuation symbols fwiw: https://drafts.csswg.org/css-text-decor/#text-emphasis-style-property.

The 3rd test is testing precisely those exceptions. Can we address them?

<img width="828" alt="image" src="https://github.com/user-attachments/assets/1805d7f7-a033-47cb-b038-45e20727dd59" />

Reply:
Yes - I know. yes, can try it in separate PR?

Reply:
Chrome doesn't implement these exceptions and the difference on that test is pretty obvious, many emphasis marks are not painted when they should be: https://wpt.fyi/analyzer?screenshot=sha1%3A647615a53bdd0ba063b31f87356b7866a3a4d92d&screenshot=sha1%3A8a8008e193af7b79b23ee4b9008b8c9354f81306

Firefox implements them properly though: https://wpt.fyi/analyzer?screenshot=sha1%3A85451987d4f4a0949dafc045df7f9402d9487f35&screenshot=sha1%3A4fc60ec3f66ef776258cb69220f91dc184151577

Reply:
> Yes - I know. yes, can try it in separate PR?

I'd prefer doing it in the same PR. It's a regression otherwise if we stop painting emphasis marks where there used to be some. Here's Mozilla's commit in case it helps: https://phabricator.services.mozilla.com/D219099

Reply:
Raised - https://bugs.webkit.org/show_bug.cgi?id=285409

Reply:
I agree with @nt1m: We shouldn't unnecessarily break them and then fix them.

Reply:
They worked until we added the punctuation exception. These are not two separate issues.

Reply:
Tackling in one PR, so I think comment is now resolved.